src/components/AdminPage/CategoryList.js

```diff
@@ -25,18 +28,41 @@ export class CategoryList extends Component {
     this.props.actions.filterByCategories(selectedCategory.length > 0 ? filteredResource : this.props.resource);
   }
 
+  clearCategoryMenuItems() {
+    // this.categoryMenuItems().map
+  }
+
   categoryMenuItems() {
     return this.props.categories.map((cat) =>
       <FormGroup key={cat} check>
         <Input type="checkbox" key={cat} onChange={() => this.handleChange(cat)} />{cat}
       </FormGroup>);
   }
+
+  handleClick = (event) => {
+    event.preventDefault();
+    console.log('hi');
```

Comment:
I would take out all the testing console messages, if you want to keep any console messages, they should be descriptive, and relevant (ex: console message saying that all the categories are loaded)